0.download_data/README.md

```diff
@@ -0,0 +1 @@
+This is a placeholding file for instructions on how to download the data.
```

Comment:
I recommend adding a bit more detail, even if this is a placeholder.

The detail I can see being:

- What are the data?
- What to expect to be in the README in the future

Reply:
updated!